lib/big_query/rollup/last_28_uniques.ex

```diff
@@ -0,0 +1,30 @@
+defmodule BigQuery.Rollup.Last28Uniques do
+  alias BigQuery.Base.Query, as: Query
```

Comment:
(You could also write this without the `as:` ... and it uses the last part of the module as the alias)